Vuex

BJordi opened the pull request:
## Summary

- Installed Vuex library.
- Added books module to the Vuex store.
- Added use of books methods, getters, and computed properties in the dashboard.

Commit:
VUEX: INSTALLED VUEX LIBRARY

Commit:
ADDED USE OF VUEX LIBRARY TO CONTAIN BOOKS LIST

- Added books module to the Vuex store.
- Added use of books methods, getters, and computed properties in the dashboard.

mcallegari10 commented on src/store.js:
```diff
@@ -1,16 +1,12 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
 
+import Books from '@/vuex/books'
```
We won't be usng a `vuex` folder. It's preferable to use a `store` folder (having this `store.js` as `index.js`) and inside, a `modules` folder with each module. The modules are imported with `camelCase`

BJordi replied:
Great!! Thanks for the explanation! :wink::+1:

Commit:
VUEX LIBRARY - FIXES IN FOLDER STRUCTURE AND AUTH HEADERS

- Rename vuex folder to store, store.js was moved inside that folder and renamed to index.js.
- Added a modules folder inside the store one, and moved the books module inside of it.

It all was made to follow our standard.
